### PR TITLE
Datetime & Human Readable issues

### PR DESCRIPTION
**Pull request by mitogh**
PR with the changes from @paulmskim & myself merged into a single PR.

### CODE REVIEW

File: src/modules/elements/time-picker/element.js [hunk 1]
```diff
@@ -184,86 +116,69 @@ export default class TimePicker extends Component {
...
+	const renderDropdownContent = ({ onClose }) => (
+		<ScrollTo>
+			{ () => (
+				<ScrollArea
+					id="tribe-element-timepicker-items"
+					key="tribe-element-timepicker-items"
+					role="menu"
+					className={ classNames( 'tribe-editor__timepicker__items' ) }
+				>
+					{ renderItem( { text: 'All Day', value: 'all-day' }, onClose ) }
```
Comment (bordoni) [inline src/modules/elements/time-picker/element.js]: Translate `All Day`

File: src/modules/elements/time-picker/element.js [hunk 2]
```diff
@@ -184,86 +116,69 @@ export default class TimePicker extends Component {
...
 }
+
+TimePicker.defaultProps = {
+	current: moment(),
+	min: undefined,
+	max: undefined,
+	start: undefined,
+	end: undefined,
+	step: HALF_HOUR_IN_SECONDS,
+	timeFormat: 'H:i',
```
Comment (bordoni) [inline src/modules/elements/time-picker/element.js]: Shouldn't we bring that from the defaults picked by the user, on the Settings page?

Comment (mitogh) [reply src/modules/elements/time-picker/element.js]: Yes, this is provided via props, however if not provided we have a default value.

Comment (mitogh) [reply src/modules/elements/time-picker/element.js]: - https://github.com/moderntribe/events-gutenberg/blob/c3a8f33a5ce713c79c8b7eb41fc7fff0d024cf62/src/modules/blocks/event-datetime/block.js#L475

- https://github.com/moderntribe/events-gutenberg/blob/c3a8f33a5ce713c79c8b7eb41fc7fff0d024cf62/src/modules/blocks/event-datetime/block.js#L513

**Pull request merged**